src/generated/types.ts

```diff
@@ -12134,7 +12136,7 @@ export interface operations {
     };
   };
   /** Gets the audit log for an enterprise. To use this endpoint, you must be an enterprise admin, and you must use an access token with the `admin:enterprise` scope. */
-  "audit-log/get-audit-log": {
+  "enterprise-admin/get-audit-log": {
```

Comment:
This is a breaking change we need to address with a deprecation in `octokit/openapi`